sympy/printing/printer.py

```diff
@@ -268,3 +271,13 @@ def _as_ordered_terms(self, expr, order=None):
             return sorted(Add.make_args(expr), key=cmp_to_key(Basic._compare_pretty))
         else:
             return expr.as_ordered_terms(order=order)
+
+
+def _print_Integer__scientific(printer, expr, limit=6, base=10, every=3):
```

Comment:
No particular reason for the double score except for indicating that it is not based on any class. Will change this. I will also add a separate test for the function.